src/confdb/data/ConfigInfo.java

```diff
@@ -5,238 +5,238 @@
 import java.util.ArrayList;
 import java.util.Collections;
 
-
 /**
- * ConfigInfo
+ * ConfigInfo 
```

Comment:
sorry for being pedantic - could you avoid adding the extra space at the end of the line ?
(here and in the other files)